tests/core/pyspec/eth2spec/test/utils.py

```diff
@@ -1,6 +1,6 @@
 from typing import Dict, Any
 from eth2spec.debug.encode import encode
-from eth2spec.utils.ssz.ssz_typing import SSZValue
+from eth2spec.utils.ssz.ssz_typing import View
```

Comment:
What exactly is a `View`?

Reply:
A `View` is a typed interpretation of a binary tree node/tree (a backing). Essentially a value, but referencing immutable data (for complex types at least, basic types are the backing themselves as well).